web/models/project.ex

```diff
@@ -23,6 +24,23 @@ defmodule Pairmotron.Project do
     |> validate_url(:url)
   end
```

Comment:
So the default changeset exists basically only to let the /admin route change whatever they want. User facing routes use only `changeset_for_create` and `changeset_for_update`